src/i18n/en.ts

```diff
@@ -499,6 +500,7 @@ const en = {
         "annotator.legend.definition.tooltip": "Definition of a known term created by a user.",
         'annotator.legend.toggle.show': 'Show legend',
```

Comment:
You might want to replace single quotes here as well

Reply:
Good point. Should simplify merging to KODI.